packages/blade/src/components/Typography/Text/Text.tsx

```diff
@@ -78,7 +78,14 @@ const getTextProps = <T extends { variant: TextVariant }>({
       props.fontSize = 75;
       props.lineHeight = 's';
     }
+    if (size === 'xsmall') {
+      props.fontSize = 25;
+      props.lineHeight = 's';
```

Comment:
are you sure the lineheight is `s` when the `fontsize` is `25` ?

Reply:
Yes it seems to be s.

<img width="572" alt="image" src="https://user-images.githubusercontent.com/35374649/210554759-2f6c0495-28a1-4d66-9611-28cbed346c05.png">

https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13708%3A180902&t=PmXXUVQ5lq9tRsca-0